tests/std/tests/GH_002992_unwrappable_iter_sent_pairs/test.cpp

```diff
@@ -0,0 +1,224 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <array>
+#include <filesystem>
+#include <forward_list>
+#include <list>
+#include <map>
+#include <set>
+#include <string>
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
+#include <xutility>
+
+#include <range_algorithm_support.hpp>
+
+using namespace std;
+
+template <class Rng>
+void test_algorithms(Rng& rng) {
+    auto dist  = ranges::distance(rng);
+    auto other = vector<ranges::range_value_t<Rng>>(static_cast<size_t>(dist));
+
+    (void) ranges::mismatch(rng.begin(), rng.end(), other.begin(), other.end());
+    (void) ranges::mismatch(other.begin(), other.end(), rng.begin(), rng.end());
+    (void) ranges::mismatch(rng, other);
+    (void) ranges::mismatch(other, rng);
+
+    if constexpr (is_lvalue_reference_v<ranges::range_reference_t<Rng>> && ranges::forward_range<Rng>) {
+        (void) ranges::uninitialized_copy(rng.begin(), rng.end(), other.begin(), other.end());
+        (void) ranges::uninitialized_copy(other.begin(), other.end(), rng.begin(), rng.end());
+        (void) ranges::uninitialized_copy(rng, other);
+        (void) ranges::uninitialized_copy(other, rng);
+
+        (void) ranges::uninitialized_copy_n(rng.begin(), dist, other.begin(), other.end());
+        (void) ranges::uninitialized_copy_n(other.begin(), dist, rng.begin(), rng.end());
+    }
+}
+
+template <class It, class Se>
+void test_unwrappable_pair() {
+    constexpr static bool is_const_unwrappable = requires(const It& ci) {

```

Comment:
We conventionally say `static constexpr`, never `constexpr static`. However, these don't need to be `static` at all. For local constants that aren't arrays, `constexpr` suffices.

Occurs below.